### DISCUSSION TITLE
I'm reviewing pending PRs.

### DISCUSSION THREAD

**Discussion by mingrammer** [mingrammer/diagrams#771] [Announcements]
I'm reviewing all the pending PRs. Sorry for the too-late review guys.

**Comment by mehdicopter** [comment]
hi, is the project dead?

**Reply by anolting** [reply]
@mingrammer There a number of people out there willing to support and asking for r/w.
Can you make a statement about your plans with this project as it is way to good to let it die.
Best Alex

**Reply by mingrammer** [reply]
We're actively maintaining it with newly joined maintainers.
